lib/how_is/pulse.rb

```diff
@@ -23,7 +23,7 @@ def text_summary
     def html_summary
       parts =
         @pulse_page_response.body
-          .split('<div class="section diffstat-summary">')
+                            .split('<div class="section diffstat-summary">')
```

Comment:
I'm actually not sure how I feel about this. My brain always initially interprets this indentation style as if `.body` and `.split` are both being called on `@pulse_page_response`, so I actually find it significantly harder to read.